web/src/user/user-settings/details/UserSettingsFlowExecutor.ts

```diff
@@ -169,21 +166,20 @@ export class UserSettingsFlowExecutor
             case ChallengeChoices.Shell:
                 return html`${unsafeHTML((this.challenge as ShellChallenge).body)}`;
             case ChallengeChoices.Native:
-                switch (this.challenge.component) {
```

Comment:
// small-switch-statement

Unlike most of the others, there's that debug _statement_ right in the middle, so I couldn't make this a conditional expression, so `if` is recommended here instead of `switch`.